browser-test/src/trusted_intermediary.test.ts

```diff
@@ -753,4 +753,233 @@ describe('Trusted intermediaries', () => {
       ])
     })
   })
+
+  describe('client list pagination', () => {
+    it('shows 1 page and no previous or next buttons when there are 10 clients', async () => {
+      const {page, tiDashboard} = ctx
+      await loginAsTrustedIntermediary(page)
+      await tiDashboard.gotoTIDashboardPage(page)
+      await waitForPageJsLoad(page)
+
+      await tiDashboard.createMultipleClients('myname', 10)
+      const cardCount = await page.locator('.usa-card__container').count()
+      expect(cardCount).toBe(10)
+
+      // No 'Previous' button
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__previous-page',
+      )
+
+      // No 'Next' button
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__next-page',
+      )
+
+      // There should be a page 1 button
+      await tiDashboard.expectPageNumberButtonOrNot('1', true)
+
+      // There should be no page 2 button
+      await tiDashboard.expectPageNumberButtonOrNot('2', false)
+
+      // The page 1 button should be the current page
+      expect(await page.innerHTML('.usa-current')).toContain('1')
+
+      // There should be no ellipses
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__overflow',
+      )
+    })
+
+    it('shows 2 pages when there are 12 clients', async () => {
+      const {page, tiDashboard} = ctx
+      await loginAsTrustedIntermediary(page)
+      await tiDashboard.gotoTIDashboardPage(page)
+      await waitForPageJsLoad(page)
+
+      await tiDashboard.createMultipleClients('myname', 11)
+
+      // Page 1 should still only show 10 clients
+      const cardCount = await page.locator('.usa-card__container').count()
+      expect(cardCount).toBe(10)
+
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
```

Comment:
suggestion (non-blocking): add a comment like you have above eg `// No "Previous" button, but there should be a "Next" button`

Reply:
Done.